src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -120,6 +120,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@SteveL-MSFT, your last commit had 12 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

ConsoleHost unit tests.-WorkingDirectory parameter.-WorkingDirectory should be processed before profiles
```powershell
Expected strings to be the same, but they were different.
Expected length: 25
Actual length:   8
Strings differ at index 0.
Expected: 'C:\Users\VssAdministrator'
But was:  'D:\a\1\s'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 736
736:                 $out[0] | Should -BeExactly (Get-Item ~).FullName
```
ConsoleHost unit tests.-WorkingDirectory parameter.Can set working directory to '~'
```powershell
Expected strings to be the same, but they were different.
Expected length: 25
Actual length:   8
Strings differ at index 0.
Expected: 'C:\Users\VssAdministrator'
But was:  'D:\a\1\s'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 700
700:             $output | Should -BeExactly $expectedPath
```
ConsoleHost unit tests.-WorkingDirectory parameter.Can set working directory to '~/77dc1fb5-b629-499e-98c0-2d82391a37d4 test'
```powershell
Expected strings to be the same, but they were different.
Expected length: 67
Actual length:   8
Strings differ at index 0.
Expected: 'C:\Users\V...'
But was:  'D:\a\1\s'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 700
700:             $output | Should -BeExactly $expectedPath
```
ConsoleHost unit tests.-WorkingDirectory parameter.Can set working directory to '~\77dc1fb5-b629-499e-98c0-2d82391a37d4 test'
```powershell
Expected strings to be the same, but they were different.
Expected length: 67
Actual length:   8
Strings differ at index 0.
Expected: 'C:\Users\V...'
But was:  'D:\a\1\s'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 700
700:             $output | Should -BeExactly $expectedPath
```
ConsoleHost unit tests.-WorkingDirectory parameter.Can use '-workingdirectory' to set working directory
```powershell
Expected strings to be the same, but they were different.
Expected length: 25
Actual length:   8
Strings differ at index 0.
Expected: 'C:\Users\VssAdministrator'
But was:  'D:\a\1\s'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 710
710:             $output | Should -BeExactly $((Get-Item ~).FullName)
```